client/me/security-2fa-backup-codes-list/index.jsx

```diff
@@ -146,14 +146,16 @@ class Security2faBackupCodesList extends React.Component {
 		let row;
 		let html = '<html><head><title>';
 
-		html += this.props.translate( 'Backup verification codes' );
+		html += this.props.translate( 'WordPress.com Backup Verification Codes' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.